lxd/main_forkproxy.go

```diff
@@ -188,9 +195,6 @@ func (c *cmdForkproxy) Run(cmd *cobra.Command, args []string) error {
 		defer file.Close()
 
 		listenerFd := file.Fd()
```

Comment:
Is this fd marked as `O_CLOEXEC`? The `Dup()` below will remove the `O_CLOEXEC` flag so that's safe. But if the `O_CLOEXEC` flag is not set on `file.Fd()` then this leaves two fds around after the `execve()`.

Reply:
It seems that somewhere along the line, `fcntl` is called with `F_DUPFD_CLOEXEC`. Does that answer your question? :)

Reply:
"somewhere along the line" doesn't give me confidence :)